src/main/java/org/stellar/sdk/responses/TransactionDeserializer.java

```diff
@@ -28,17 +28,28 @@ public TransactionResponse deserialize(JsonElement json, Type typeOfT, JsonDeser
     if (memoType.equals("none")) {
       memo = Memo.none();
     } else {
-      String memoValue = json.getAsJsonObject().get("memo").getAsString();
+      // Because of the way "encoding/json" works on structs in Go, if transaction
```

Comment:
Could you elaborate more on why java code requires changes due to go behavior?  This strikes me as a bug in the behavior in our go implementation.

Reply:
https://github.com/stellar/go/issues/149